### PR TITLE
Move getCurrentTime() to database.js

### PR DESCRIPTION
**Pull request by kaysabelle**
RIP Resources.js
2015-2015

### CODE REVIEW

**Comment by MinickLS** [comment]
I was behind you when you made the pull request. Pass.

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/Resources.js [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-		var timeNow = new Date();
-		var hours   = timeNow.getHours();
-		var minutes = timeNow.getMinutes();
-		var seconds = timeNow.getSeconds();
-		var timeString = "" + ((hours > 12) ? hours - 12 : hours);
-		timeString  += ((minutes < 10) ? ":0" : ":") + minutes;
-		timeString  += ((seconds < 10) ? ":0" : ":") + seconds;
-		timeString  += (hours >= 12) ? " P.M." : " A.M.";
-		return timeString;
-	}
```
Comment (jeslin) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/Resources.js:15]: RIP

**Comment by kaysabelle** [comment]
Ew what a creeper. Reject the pass

**Comment by vinchan** [comment]
Either this change isn't on RL Docs yet or this doesn't follow the RL docs specs.
What we agreed on before in the RL Docs is that Resources.js will still exist, but only be called on by internal functions. The reasoning for this is that we agreed on directConnect returning a parameter called "timeStamp", along with the time. The plugin should not return a timeStamp that is dependent on an app/user specific function like one in database.js. Now if we are saying that directConnect's timeStamp should no longer exist, then this change is fine.

**Pull request merged**